Indonesia/indo.md

```diff
@@ -0,0 +1,5 @@
+Petualangan kamu dimulai. Kamu menyadari banyak jalan yang bisa kamu tempuh.
+Jalan mana yang akan kamu ambil? Hanya kamu yang bisa menentukan.
+
+
+Kamu bosan di rumah, sehingga kamu memutuskan untuk [jalan - jalan](jalan%20-%20jalan/jalan.md).
```

Comment:
The link is not broken, i already tested it and it works. I added space to the folder name, so when it translated to url, it will encoded like that